src/homework17/ThreadSafeList.java

```diff
@@ -0,0 +1,26 @@
+package homework17;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+//0.1. Створити клас ThreadSafeList. Клас імплементрізує потокобезпечну колекцію.
+//0.1 Реалізувати методи: add, remove, get
+//0.2 Методи повинні бути реалізовані з урахуванням того, що вони будуть використовуватися в конкурентному середовищі.
+
+
+public class ThreadSafeList {
```

Comment:
The class must implement the List interface in a thread-safe manner. Methods add, get and remove must be implemented. The rest of the methods can be empty/not implemented (for example, throw OperationNotSupportedException).